proto/poktroll/application/event.proto

```diff
@@ -19,6 +19,8 @@ enum ApplicationUnbondingReason {
 // EventApplicationStaked is emitted when an application is staked or up-staked.
 message EventApplicationStaked {
   poktroll.application.Application application = 1 [(gogoproto.jsontag) = "application"];
+  // The end height of the session in which the transfer began.
```

Comment:
```suggestion
  // The end height of the session in which the staking occurs.
```

Reply:
Not sure how this diff got here. I've ported this comment to #867: https://github.com/pokt-network/poktroll/pull/867/files#r1806125103